pages/breeds/index.js

```diff
@@ -3,44 +3,118 @@ import Thumbnail from "@/components/Breeds/Thumbnail";
 import Link from "next/link";
 import { styled } from "styled-components";
 
+import { useEffect, useState } from "react";
+import {
+  dogBreedGroups,
+  dogBreedHeight,
+  dogBreedTemperament,
+} from "@/data/dogBreedFilter";
+import { filterDogBreeds } from "@/utils/filterDogBreeds";
+import Wrapper from "@/components/Form/Wrapper.styled";
+import StyledForm from "@/components/Form/Form.styled";
+import StyledList from "@/components/List.styled";
+import StyledLink from "@/components/ListLink.styled";
+import Select from "@/components/Form/Select.styled";
+
+const initialFilter = {
+  group: "all",
+  temperament: "all",
+  size: "all",
+};
+
 export default function BreedList({ dogData }) {
+  const [filter, setFilter] = useState(initialFilter);
+  const [showBreeds, setShowBreeds] = useState(dogData);
```

Comment:
here the naming is confusing. What exactly is dogData? the state naming suggests that it is a boolean, the prop name that it is an object.

Reply:
changed the names